src/res/locales/app.dev.po

```diff
@@ -233,6 +233,10 @@ msgstr "Never"
 msgid "settings_notify_mute"
 msgstr "Mute sounds"
 
+#: Hide unread message count
+msgid "settings_notify_count"
```

Comment:
Since the message counts are not notifications of sorts, thise should probably be renamed to `settings_show_message_counts`